server/src/packages/landuse-form/landuse-form.attrs.ts

```diff
@@ -448,5 +448,6 @@ export const LANDUSE_FORM_ATTRS = [
   'dcp_landuse_dcp_sitedatageneralform_landuseform',
   'dcp_landuse_Annotations',
   'dcp_dcp_landuse_dcp_landuseaction',
-  'dcp_dcp_landuse_dcp_sitedatahform_landuseform'
+  'dcp_dcp_landuse_dcp_sitedatahform_landuseform',
+  '_dcp_project_value'
```

Comment:
Check this **_**dcp_project_value to see if this should be dcp_dcp_project_value or dcp_project_value